.github/workflows/test.yml

```
@@ -0,0 +1,36 @@
+name: Tests
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)